test/letters/01-authed.e2e.spec.js

```diff
@@ -83,6 +83,43 @@ module.exports = E2eHelpers.createE2eTest(
       .click('div.step-content > p:nth-child(3) > a') // link to go back to confirm-address
       .expect.element('.city-state').to.be.present.before(Timeouts.normal);
 
+    // -- Go to letters list -- //
+
+    client
+      .click('.view-letters-button')
+      .assert.urlContains('/letters/letter-list')
+      .waitForElementVisible('.step-content', Timeouts.normal)
+      .click('.step-content div.form-review-panel:nth-of-type(4) button') // open the bsl accordion
+      .waitForElementPresent('#militaryService', Timeouts.normal);
+
+    // poke all the checkboxes and expect them to all be unselected
+    client
+      .expect.element('#militaryService').to.have.attribute('checked').equals('true');
+    client
+      .click('#militaryService')
+      .expect.element('#militaryService').to.not.have.attribute('checked');
+    client.execute(() => {
+      return Array.from(document.querySelectorAll('#benefitInfoTable input[type="checkbox"]'))
+        .map(e => e.getAttribute('id'));
+    }, [], (obj) => {
```

Comment:
Interesting . . . this callback receives an object that includes the value returned from body param function.

Reply:
It's a fun, undocumented feature. 😏 
'Course, I say that, but it probably is documented somewhere. I just had to `console.log` and experiment to figure out how to make it work.